dashboards/hbase/metadata.yaml

```diff
@@ -15,3 +15,19 @@ sample_dashboards:
     related_integrations:
       - id: hbase
         platform: GCE
+  -
+    category: HBase
+    id: hbase-prometheus-overview
+    display_name: Hbase Prometheus Overview
+    description: "This dashboard is based on metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including Region Servers, Dead Region Servers, Read Requests, and Write Requests."
```

Comment:
Same comment as before about the exporter used for scraping Hbase metrics. Is it https://github.com/prometheus/jmx_exporter?

Reply:
That is correct.